runtime/tests/port/j9vmemTest.c

```diff
@@ -2298,22 +2368,35 @@ j9vmem_testFindValidPageSize_impl(struct J9PortLibrary *portLibrary, char *testN
 		expectedPageFlags = defaultLargePageFlags;
 		expectedIsSizeSupported = FALSE;
 	} else {
-		expectedPageSize = FOUR_KB;
+		expectedPageSize = defaultPageSize;
 		expectedPageFlags = J9PORT_VMEM_PAGE_FLAG_NOT_USED;
 		expectedIsSizeSupported = FALSE;
 	}
 
-	verifyFindValidPageSizeOutput(portLibrary, testName,
-								expectedPageSize, expectedPageFlags, expectedIsSizeSupported,
-								requestedPageSize, requestedPageFlags, isSizeSupported);
+	verifyFindValidPageSizeOutput(
+			portLibrary, testName,
+			expectedPageSize, expectedPageFlags, expectedIsSizeSupported,
+			requestedPageSize, requestedPageFlags, isSizeSupported);
 
 
-	/* Test -Xlp:codecache options */
+	/* Test -Xlp:codecache options. */
+
+	/* For 32-bit executable pages, honour the request if
+	 * - request is for 64K pages, or
+	 * - request is for 16M pages AND CodeCacheConsolidation is enabled
+	 * otherwise, fallback to default (as determined below).
```

Comment:
```suggestion
	/* For 32-bit executable pages, honour the request if:
	 * - the request is for 64K pages, or
	 * - the request is for 16M pages and CodeCacheConsolidation is enabled;
	 * otherwise, fallback to default (as determined below).
```